Tracking Plans update (#531)
rudderlabs/rudderstack-docs: data-governance/tracking-plans/index.mdx

Tracking Plans let you proactively monitor and act on non-compliant event data coming into your RudderStack sources based on predefined plans. This can help you prevent or de-risk situations where missing or improperly configured event data can break your downstream destinations.

<div class="warningBlock">
  

> The Tracking Plans are supported only for the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/"><code class="inline-code">track</code></a> events sent to the <strong>warehouse destinations</strong>.

Author: ```suggestion
The Tracking Plans currently support only the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/"><code class="inline-code">track</code></a> events sent to the <strong>warehouse destinations</strong>.
```

Tracking Plans let you proactively monitor and act on non-compliant event data coming into your RudderStack sources based on predefined plans. This can help you prevent or de-risk situations where missing or improperly configured event data can break your downstream destinations.

<div class="warningBlock">
  

> The Tracking Plans currently support only <a href="https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/" target="_blank"><code class="inline-code">track</code></a> events and are applicable only for <strong>warehouse destinations</strong>.

Author: ```suggestion

The Tracking Plans are supported only for the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/"><code class="inline-code">track</code></a> events sent to the <strong>warehouse destinations</strong>.
```

    </a>
</div>

## FAQ

### Which calls are supported for the RudderStack Tracking Plans?

The Tracking Plans feature currently supports only [`track`](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/track/) events.

### Are the Tracking Plans supported for all the destinations?

> No - the Tracking Plans feature is currently supported only for the [warehouse destinations](https://rudderstack.com/docs/data-warehouse-integrations/)

Reviewer: ```suggestion
No - the Tracking Plans feature is currently supported only for the [warehouse destinations](https://rudderstack.com/docs/data-warehouse-integrations/).
```
